osu.Game/Online/API/IAPIProvider.cs

```diff
@@ -134,6 +134,11 @@ public interface IAPIProvider
         /// </summary>
         void UpdateStatistics(UserStatistics newStatistics);
 
+        /// <summary>
+        /// Update the friends status of the current user.
+        /// </summary>
+        void UpdateLocalFriends();
```

Comment:
Related to preceding comment - if the need to refetch users can be removed, this method probably doesn't need to exist. (I guess there is https://github.com/ppy/osu/issues/13604, but it's (a) out of scope here, and (b) not even a given that a poll like the OP of that proposes is the right thing to do.)

Reply:
Ok, I'll keep him for now until the results come in?